core/commands/active.go

```diff
@@ -9,31 +9,41 @@ import (
 	"time"
 
 	cmds "github.com/ipfs/go-ipfs/commands"
+	e "github.com/ipfs/go-ipfs/core/commands/e"
+
+	"gx/ipfs/QmPMeikDc7tQEDvaS66j1bVPQ2jBkvFwz3Qom5eA5i4xip/go-ipfs-cmdkit"
 )
 
 var ActiveReqsCmd = &cmds.Command{
-	Helptext: cmds.HelpText{
+	Helptext: cmdkit.HelpText{
 		Tagline: "List commands run on this IPFS node.",
 		ShortDescription: `
 Lists running and recently run commands.
 `,
 	},
 	Run: func(req cmds.Request, res cmds.Response) {
-		res.SetOutput(req.InvocContext().ReqLog.Report())
+		ctx := req.InvocContext()
```

Comment:
this change is unecessary, no?

Reply:
Oh yes, you're right. I think that line hit a panic at some time and I wanted to know which part was causing it.